L1Trigger/L1TTrackMatch/src/Cordic.cc

```diff
@@ -104,7 +104,7 @@ EtMiss Cordic::toPolar(Et_t x, Et_t y) const {
 
   // emulate fw rounding with float division then floor
   float tempMET = (float)(x * magNormalisationLUT[cordicSteps - 1] * kMaxTrackPt) / ((float)kMaxMET);
-  ret_etmiss.Et = (int)floor(tempMET) >> (mMagnitudeBits + TTTrack_TrackWord::TrackBitWidths::kRinvSize - kMETSize);
+  ret_etmiss.Et = tempMET / pow(2,(mMagnitudeBits + TTTrack_TrackWord::TrackBitWidths::kRinvSize - kMETIntSize + (kMETSize - kMETIntSize)));
```

Comment:
I don't think this is quite right. In my testing of this, this means that `tempMET` is divided by `2^25`. However, to return the same fixed point value as before, it needs to be divided by `2^20`.

Reply:
Fixed, realised I hadn't updated a scaling factor that is used in the ntuple maker making the result look fine in the ntuples